clickbench/benchmark.sh

```diff
@@ -1,11 +1,16 @@
 #!/bin/bash
 set -e
 
-mode="${1:-single}" # single/multi
-sweep_cores="${2:-single}" # single/multi
+# single = single parquet file (hits.parquet)
+# multi = multiple parquet files (100 files in hits_multi)
+mode="single"
```

Comment:
I actually think it is fine to use the multi-file dataset consistently (which is what I wrote in the paper) so updating that here

```suggestion
# Use the multi-file dataset
mode="multi"
```